Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -1466,6 +1476,16 @@ end
 					},
 					Notes = "Returns true if the client has registered to receive messages on the specified plugin channel.",
 				},
+				IsModded =
+				{
+					Returns =
+					{
+						{
+							Type = "boolean",
+						},
+					},
+					Notes = "Returns true if the client is modded with Forge.",
```

Comment:
Should this be Forge-specific, or (future-proofing) would this mean any client-side mod? Perhaps consider renaming to indicate Forge-specific.